src/colmap/estimators/bundle_adjustment.h

```diff
@@ -287,6 +299,51 @@ class RigBundleAdjuster : public BundleAdjuster {
   std::unordered_set<double*> parameterized_quats_;
 };
 
+class PosePriorBundleAdjuster : public BundleAdjuster {
+ public:
+  struct Options {
+    // Whether to use prior camera positions
+    bool use_prior_position = true;
+
+    // Maximum RANSAC error for Sim3D alignment
+    double ransac_max_error = 0.;
+
+    // Sim3d transformation that project reconstruction's centroid to (0.,0.,0.)
+    Sim3d sim_to_center;
+  };
```

Comment:
Unlike `RigBundleAdjuster::Options`, this looks like an internal state of the object - is this correct? then 1) should this really be named `Options`? 2) shouldn't the definition be private since there is no public getter/setter for `prior_options_`?

Reply:
The struct Options is now used to store the use of a robust loss on the priors & the related scale as well as the ransac threshold error.  Other parameters have been moved into the class directly.